### PR TITLE
Add missing musl utmpx.h constants

### PR DESCRIPTION
**Pull request by andrewliebenow**

### CODE REVIEW

**Comment by andrewliebenow** [comment]
See https://git.musl-libc.org/cgit/musl/tree/include/utmpx.h?h=v1.1.24.

I'm not sure if adding these constants would be considered a breaking change (https://doc.rust-lang.org/cargo/reference/semver.html#item-new).

File: libc-test/semver/linux-musl.txt [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # TODO: musl.
+# linux-gnu.txt is alphabetized, but this file is not
```
Comment (tgross35) [inline libc-test/semver/linux-musl.txt:2]: Sorted in https://github.com/rust-lang/libc/pull/3954, could you rebase once that merges and sort these in?

Comment (andrewliebenow) [reply libc-test/semver/linux-musl.txt]: Done

**Review by tgross35 — APPROVED** [review]
Thanks!

**Pull request merged**